baselines/dom.generated.d.ts

```diff
@@ -5455,7 +5455,7 @@ interface GlobalEventHandlers {
      * Fires when an error occurs during object loading.
      * @param ev The event.
      */
-    onerror: ErrorEventHandler;
+    onerror: OnErrorEventHandler;
```

Comment:
Shouldn't this be `OnErrorEventHandler | null`?

Reply:
Ah, sorry, was just looking at the patch and didn't notice the different names.  Does the lib generator have test cases somewhere? Would be nice to be able to add a null assignment somewhere to be sure it compiles.

Reply:
Unfortunately we currently don't have one, but definitely great to have.